src/lightkurve/periodogram.py

```diff
@@ -507,6 +508,67 @@ def __div__(self, other):
     def __rdiv__(self, other):
         return self.__rtruediv__(other)
 
+    def peak_list(self, powerlimit=None):
+        """Lists in descending order the peaks found in the periodogram with 
+        the find_peaks function, which calculates the Prominence of the peak,
+        and Lower & Upper Half Width at Half Maximum (HWHM) of this Prominence.
+        For more information on this SciPy signal function, 
+        please follow the link to their documentation:
+        https://docs.scipy.org/doc//scipy/reference/generated/scipy.signal.find_peaks.html
+        Additionally final column of the list shows the ratio of the peak with
+        the top peak, where deviation from neat fractions (like 2/1, 1/2, 3/2,
+        2/3, 4/3, 3/4, etc.) is an indication of another candidate.
+
+        Parameters
+        ----------
+        powerlimit : number or ndarray or sequence, optional
+            Required power of peaks. Either a number, None, an array matching x 
+            or a 2-element sequence of the former. The first element is always 
+            interpreted as the minimal power and the second, if supplied, as the 
+            maximal required power. By default or None, 5% of the periodogram's 
+            max_power value will be used as the number for the powerlimit.
+
+        Returns
+        -------
+        Table : `astropy table` object
+            Returns a Table object extracted from the periodogram.
+        """
+        if self.default_view == 'period':
+            view = self.period
+            x = 'Periodicity'
+            y = '.1f'
+        elif self.default_view == 'frequency':
+            view = self.frequency
+            x = 'Frequency'
+            y = None
+        if powerlimit is None:
+            powerlimit = float(self.max_power)/20
```

Comment:
This line raised Error if the periodgram'x max_power has units such as electron/s (e.g., if the source lightcurve is not normalized).